.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       NEXT_PUBLIC_APPWRITE_API_URL: ${{ secrets.NEXT_PUBLIC_APPWRITE_API_URL }}
       NEXT_PUBLIC_APPWRITE_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_APPWRITE_PROJECT_ID }}
       NEXT_PUBLIC_APPWRITE_DATABASE_ID: ${{ secrets.NEXT_PUBLIC_APPWRITE_DATABASE_ID }}
+      APPWRITE_API_KEY: ${{ secrets.APPWRITE_API_KEY }}
     strategy:
```

Comment:
With the `secrets.APPWRITE_API_KEY`, will the .env file be updated and will you share the new .env file with us? :) Please :)

Reply:
Yup! I definitely will :)